doc/releases/migration-guide-3.6.rst

```diff
@@ -358,6 +358,9 @@ Device Drivers and Device Tree
 
   (:github:`66427`)
 
+* The :dtcompatible:`st,hci-spi-v1` and :dtcompatible:`st,hci-spi-v2` drivers are introduced
+  to provide BLE support for BlueNRG-based devices according to their SPI protocol version.
```

Comment:
This entry doesn't seem necessary? Migration guide entries are for things that have changed and may break people's applications. In this case the bindings being new, AFAICT, people shouldn't really have anything to migrate?

Reply:
We could put them under the recommended changes instead?

Reply:
Right, looking at the PR that introduced the change it looks like it might be the case indeed. @HoZHel, would it make sense (i.e. is the change not 100% mandatory for folks, but only a "nice-to-do?)? In any case, I'd like this bullet point to be more prescriptive regarding what people should actually do, i.e. it looks like people should add the compatible to their existing `zephyr,bt-hci-spi` node?

BTW @henrikbrixandersen I'd say the "recommended change" about the ST sensor DT macro should be removed. It doesn't seem like it's describing a "you'd better migrate this thing now before it becomes a *required* change in the future" situation, is it's basically just a new (albeit nice) convenience tool.

Reply:
I think migrating existing dts file for BlueNRG based boards seems mandatory to me.